glide.lock

```diff
@@ -68,19 +70,19 @@ imports:
   - require
   - suite
 - name: github.com/uber-go/atomic
-  version: 845920076a298bdb984fb0f1b86052e4ca0a281c
+  version: 12f27ba2637fa0e13772a4f05fa46a5d18d53182
```

Comment:
these are probably innocuous but dependency updates should be in separate PRs (unless they are required here).

Reply:
Sorry for that,  it was autoupdated when I run glide get .